server/scsynth/scsynth_main.cpp

```diff
@@ -95,7 +95,7 @@ void Usage()
 		"             messages from Poll.\n"
 		"          The default is 0.\n"
 #ifdef _WIN32
-		"   -U <ugen-plugins-path>\n"    
+		"   -U <ugen-plugins-path>\n"
```

Comment:
if you do redo this branch get rid of this whitespace change and please avoid changes like this in the future. it makes the diff noisier which makes it harder to review and also fragments git-blame history